### PR TITLE
Data dictionary

### PR DESCRIPTION
**Pull request by bwalsh**
*** Work in progress ***

This PR:

* implements new data model - see datadictionary/
* updates documentation - see aced.md/
* updates portal-service - see examples/Secrets/gitops.json

### CODE REVIEW

File: docs/aced.md [hunk 1]
```diff
@@ -463,6 +463,44 @@ environment:
...
+@@ -125,7 +125,7 @@ services:
+       - ./scripts/peregrine_setup.sh:/peregrine_setup.sh
+       - ./datadictionary/gdcdictionary/schemas:/schemas_dir
+     environment: &env
+-      DICTIONARY_URL: https://s3.amazonaws.com/dictionary-artifacts/datadictionary/develop/schema.json
++      DICTIONARY_URL: https://aced-public.s3.us-west-2.amazonaws.com/aced.json
+
+```
+
+* add custom graphql section to
```
Comment (lbeckman314) [inline docs/aced.md:495]: Where should the custom graphql section be added to?

**Review by lbeckman314 — APPROVED** [review]
- Reviewed [uc-cdis](https://github.com/uc-cdis/compose-services/blob/master/docs/using_the_commons.md#changing-the-data-dictionary) and [Gen3](https://gen3.org/resources/user/dictionary/) documentation on data models.
- Ran `compile.py` against the `yaml` files in schemas and outputted the expected the `aced.json` file.

File: docs/aced.md (hunk 1 above)
Comment (bwalsh) [reply docs/aced.md:495]: See https://github.com/ACED-IDP/compose-services-training/pull/8/commits/9472acf9229bff6cb2fa14b81d7ae5312304529f#diff-974fe187f1de7376743d2cfebeeebe63729d4d409bbe7077d048e185dacebba3R509

Comment (bwalsh) [reply docs/aced.md:495]: see examples/Secrets/gitops.json

**Pull request merged**